hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -131,6 +131,14 @@
       tags = ConfigTag.CLIENT)
   private int maxECStripeWriteRetries = 10;
 
+  @Config(key = "exclude.nodes.expiry.time",
+      defaultValue = "600000",
+      description = "Ozone EC client to remove the node from the exclude" +
```

Comment:
Should the expiry apply to both EC and non-EC clients?

The description doesn't read nicely for me - can we reword to something like:

> Time after which an excluded node is reconsidered for writes. If the value is zero, the node is excluded for the life of the client